lnbits/core/migrations.py

```diff
@@ -301,3 +301,52 @@ async def m010_create_installed_extensions_table(db):
         );
     """
     )
+
+
+async def m011_add_deleted_to_wallets(db):
+    """
+    Adds deleted column to wallets.
+    """
+    try:
+        await db.execute("ALTER TABLE wallets ADD COLUMN deleted BOOLEAN")
```

Comment:
- it is much simpler not to allow NULL: `BOOLEAN NOT NULL DEFAULT false`
  - if necessary the value will be set to `true` in `m012_set_deleted_wallets`
  - the above condition `AND deleted is NULL OR deleted = false` will reduce to `AND deleted = false`